draft-ietf-acme-acme.md

```diff
@@ -226,9 +224,9 @@ and make it available to the client.
                                     <-------             Certificate
 ~~~~~~~~~~
 
-To revoke a certificate, the client simply sends a revocation request indicating
-the certificate to be revoked, signed with an authorized key pair. The server
-indicates whether the request has succeeded.
+To revoke a certificate, the client sends a revocation request indicating
+the certificate to be revoked, signed with the account's private key, or with
```

Comment:
I like your new wording more, but I think there's one aspect lost in translation.

The text "signed with an authorized key pair" allows for the notion that I can create a 2nd account keypair, authorize all the same domains and have that new account keypair considered an "authorized key pair" for revoking a certificate with the same set of names. 

This is a pretty clunky thing to express but I think the new version makes it sound like only the original issuing account keypair or the certificate's private key could be used for revocation.

Am I misunderstanding?